packages/attribute-registry-process/test/integration/createCertifiedAttribute.test.ts

```diff
@@ -102,20 +159,42 @@ describe("certified attribute creation", () => {
     };
 
     await addOneTenant(tenant);
-    await addOneAttribute(attribute);
+    await addOneAttribute({
+      ...attribute,
+      name: attribute.name.toUpperCase(),
+      code: attribute.code.toUpperCase(),
+      origin: getTenantOneCertifierFeature(tenant).certifierId,
+    });
+    expect(
+      attributeRegistryService.createCertifiedAttribute(
+        {
+          name: attribute.name,
+          code: attribute2.code,
```

Comment:
For readability reasons I would create a const with name and code shared between two attribute, and using it as input and expected.

```javascript
const attributeName = "${mockAttribute.name}-test"
const attributeCode = "123456cd"
```

```suggestion
          name: attributeName,
          code: attributeCode,
```

Reply:
Fixed as suggested, thanks! 👍